app/config/constants/index.js

```diff
@@ -12,6 +12,7 @@ export default {
   maps: {
     lat: 27.568640,
     lng: -33.461281,
+    basemap: 'https://api.mapbox.com/styles/v1/forestwatcher/cj4772tx20vsv2rrza5q3v0d4/tiles/256/{z}/{x}/{y}@2x',
```

Comment:
I remember going through this with @juancarlosalonso, the map displays in standard definition by default, but we can load the retina map if we remove the 256 from `/tiles/256/{z}/{x}/{y}@2x'`. I suppose as an improvement we could conditionally add/remove this value. For now, I think the best solution is to load the one that looks better, given that we intend to cache it, and later on add the conditional definition logic.